lib/thrift/parser/file_group.ex

```diff
@@ -57,19 +62,29 @@ defmodule Thrift.Parser.FileGroup do
                 resolutions: resolutions}
   end
 
-  def add_includes(%__MODULE__{} = group,
-                   %ParsedFile{schema: schema, file_ref: file_ref}) do
+  defp add_includes(%FileGroup{} = group, %ParsedFile{schema: schema, file_ref: file_ref}) do
+    # Search for included files in the current directory (relative to the
+    # parsed file) as well as any additionally configured include paths.
+    include_paths = [Path.dirname(file_ref.path) | group.include_paths]
 
-    Enum.reduce(schema.includes, group, fn(include, file_group) ->
-      parsed_file = file_ref.path
-      |> Path.dirname
-      |> Path.join(include.path)
-      |> FileRef.new
-      |> ParsedFile.new
-      add(file_group, parsed_file)
+    Enum.reduce(schema.includes, group, fn(include, group) ->
+      parsed_file =
+        include.path
```

Comment:
Indenting a pipeline below the assignment/match? Yes, I started noticing this is how `vim-elixir` wanted to format my code, and then I browsed around the Elixir standard library and found the same pattern. I've grown to find it pretty readable, too.